readme.md

```diff
@@ -1,5 +1,113 @@
+# .NET Foundation Project Committee
 
-This repo is also used to provide support for existing projects.
+## Background
+
+The .NET Foundation, an independent 501(C)(6) corporation, is chartered to foster open development and collaboration around the growing collection of open source technologies for .NET. It serves as a forum for commercial and community developers alike to strengthen the future of the .NET ecosystem by promoting openness, community participation and rapid innovation.
+
+The mission of the .NET Foundation is to foster a community of .NET developers focused on collaborating on commercially friendly open source software projects that enhance the .NET platform and create value within the .NET ecosystem.
+
+## Rationale
+
+An important aspect of the .NET Foundation's vision is to attract and foster vibrant communities of open source projects and contributors. However, in order to preserve the relevancy and credibility of the .NET Foundation's brand and reputation, it must ensure that all member projects fully embrace its mission and values and are willing to function as first class ambassadors and representatives of the .NET Foundation. It is the role of the Project Committee to assist the Board of Directors in ensuring this alignment.
+
+## Responsibilities
+
+The Project Committee manages the day to day requirements of Projects. This includes but is not limited to the following:
+
+- Review and evaluate new Project applications
+- Monitoring health of existing Projects
+- Handling the retirement of Projects
+- Providing support to Projects.
+
+## Organizational Structure
+
+The Project Committee is comprised of volunteer .NET Foundation members and has a designated Chairperson appointed by the Executive Director. The Chairperson ensures that the Project Committee meets monthly to discuss and review items pertaining to Projects and provides information and recommendations to the Executive Director so that they can be presented to the Board of Directors for approval. All official communication from the .NET Foundation pertaining to Projects must be delivered by the Executive Director.
+
+## Evaluation Criteria
+
+Transparency is a key open source characteristic, and as a result the criteria for evaluating .NET Foundation projects is designed to be as measurable and objective as possible. The criteria is not only important for evaluating new project applications but for measuring the ongoing health of existing projects. There are two types of criteria which have been defined:
+
+**Eligibility** - this criterion is binary in nature and is used to determine if a project is fully aligned with the mission and values of the .NET Foundation.
+
+**Health** - this criterion is quantitative in nature and is based on publicly accessible metrics which can be used to evaluate the health and maturity of a project.
```

Comment:
`Maturity` is still a little bit of a hang-up for me. It tends to be highly contextualized / debated, is easy to misconstrue, and almost always has some sort of organizational or communal politics attached to it.

As an alternative, I've been exploring the idea of confidence meters. And rather than specifying a requirement / target for such metrics, it's instead about transparency.

Confidence meters are transparent by nature. Confidence is from one group's perspective, but is not the law. Confidence is something that can be explained and others can choose to feel similarly based on the explanation.

Another key of a confidence meters: it doesn't make the Foundation responsible for getting the assessment correct, which also allows the foundation to be less prescriptive in its modeling.

So rather than health requirements, we could suggest that any .net member project will have a transparent confidence meters across consistent categories/areas.

Think about this manifesting itself on a scale of one to five, where five is very confident -- examples of rough draft items could include:

* The authors take appropriate security considerations and safeguard secrets and data as appropriate.
* Users will be aware of breaking changes in a standardized way.
* New changes to software will not endanger old functionality.
* There is a path for others to contribute to this project.
* The project is actively maintained

Etc.

Reply:
Although the term "maturity" is used in a couple places in this document, the focus is on "health" - ie. there is Eligibility Criteria and Health Criteria.